docs/TROUBLESHOOTING.md

```diff
@@ -64,6 +64,13 @@ the kubelet logs, you can execute:
 ```shell script
 $ oc adm node-logs -l kubernetes.io/os=windows --path=/kubelet/kubelet.log
 ```
+
+## How to collect `containerd` runtime logs
```

Comment:
i'd drop the code formatting here as it is not a command

Reply:
Agree. Fixed in https://github.com/openshift/windows-machine-config-operator/pull/1001/commits/ab44481eb4dd4d778b99110c78b398ea95c08419#